doc/userguide/ug2html.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```

Comment:
This should be changed to `python3`. Otherwise running the script like `doc/userguide/ug2html.py` will typically run it on Python 2.